### PR TITLE
Adjust JWT example to use RSA acceleration

### PR DESCRIPTION
**Pull request by tzerrell**
Makes use of the RSA acceleration from https://github.com/risc0/risc0/pull/2481 and https://github.com/risc0/RustCrypto-RSA/pull/1 to accelerate the JWT example.

Note that I had to adjust the risc0 crate dependencies of this example to point at github rather than including them via relative path; this is because the [RSA patch](https://github.com/risc0/RustCrypto-RSA/pull/1) this uses depends on a pinned commit of risc0, and if I don't use a github dependency here, I end up with errors like
```
error: the `#[global_allocator]` in risc0_zkvm_platform conflicts with global allocator in: risc0_zkvm_platform
```

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -326,6 +326,9 @@ jobs:
       - name: test in dev mode
         run: RISC0_DEV_MODE=1 cargo test --locked -F $FEATURE --workspace --exclude prover-example
         working-directory: examples
+      - name: test JWT validator separately
+        run: RISC0_DEV_MODE=1 cargo test --locked -F $FEATURE
+        working-directory: examples/jwt-validator
       - name: test prover example
```
Comment (flaub) [inline .github/workflows/main.yml:332]: we'll need to explicitly exclude this from the line above if we're going to do it separately, but why do we need to do this?

File: examples/Cargo.toml [hunk 2]
```diff
@@ -16,8 +16,6 @@ members = [
   "hello-world",
   "json",
   "json/core",
-  "jwt-validator",
-  "jwt-validator/core",
```
Comment (flaub) [inline examples/Cargo.toml:20]: Why do we need this?

File: examples/jwt-validator/Cargo.toml [hunk 3]
```diff
@@ -3,12 +3,17 @@ name = "jwt-validator"
 version = "0.1.0"
 edition = "2021"
 
+[workspace]
+
 [dependencies]
 jwt-core = { path = "core" }
 jwt-methods = { path = "methods" }
-risc0-zkvm = { path = "../../risc0/zkvm" }
+risc0-zkvm = { git = "https://github.com/risc0/risc0" }
```
Comment (flaub) [inline examples/jwt-validator/Cargo.toml:11]: why do we do this?

**Comment by tzerrell** [comment]
Will not work, replaced by #2488

**Pull request closed without merging**